get-started/index.md

```diff
@@ -2,7 +2,7 @@
 # Feel free to add content and custom Front Matter to this file.
 # To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
 
-layout: guides
+layout: get-started
```

Comment:
Seems to be some issues with css styles.

![Screenshot from 2023-04-28 15-50-25](https://user-images.githubusercontent.com/68053619/235240096-3bc3f816-252f-4fe7-8ba5-0ccb1dd86826.png)

```
     Build Warning: Layout 'get-started' requested in get-started/index.md does not exist.
                    ...done in 0.408386187 seconds.
                    
      Regenerating: 1 file(s) changed at 2023-04-28 15:49:28
                    get-started/index.md
       Jekyll Feed: Generating feed for posts
```

Reply:
Haizz nvmind things keep getting cached again :))